device/globals/game.gd

```diff
@@ -33,7 +33,7 @@ func mouse_enter(obj):
 	# When following the mouse, prevent text from flashing for a moment in the wrong place
 	if ProjectSettings.get_setting("escoria/ui/tooltip_follows_mouse"):
 		var pos = get_viewport().get_mouse_position()
-		tooltip.set_position(pos)
+		tooltip.set_position(pos - (tooltip.get_size() / Vector2(2, 1)))
```

Comment:
Looks good, but could it be written as `tooltip.position pos - tooltip.get_size() / Vector2(2, 1)` to be a bit easier to parse mentally?

Reply:
Afraid not. `"Invalid set index 'position'`

Sometimes I don't understand when the setter function needs to be called and when assignments work.

Reply:
That actually made it harder for me to read, though I know it has precedence, so I put the subtraction on the line above and made it simply set the new position. Hope that's fine ;)